examples/model_selection/plot_gp_search.py

```diff
@@ -0,0 +1,69 @@
+"""
+==================================================================
+Plotting Performance of different hyperparameter selection methods
+==================================================================
+
+In this plot you can see the validation scores of a Ridge regression model
+along steps of different hyperparameter selection methods.
+"""
+print(__doc__)
+
+import matplotlib.pyplot as plt
+
+import numpy as np
+from scipy import stats
+
+from sklearn.model_selection import GPSearchCV
+from sklearn.model_selection import GridSearchCV
+from sklearn.model_selection import RandomizedSearchCV
+from sklearn.linear_model import Ridge
+from sklearn.datasets import make_regression
+
+# Loading the Digits dataset
+X, y = make_regression(
+    n_samples=50, n_features=500, noise=1., random_state=0)
+
+# build a classifier
+clf = Ridge()
+
+# specify parameters and distributions to sample from
+expon = stats.expon(scale=20)
+expon.random_state = 0  # for reproductible results
```

Comment:
Thanks @glouppe , the example now works fine without the need to set random_state in the rv